### PR TITLE
Fix: Fixed crash that would occur when editing file permissions

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

- Re-enable the editing
- Code the functionalities, such as momdifying and deleting specific access control
- Add the editor implemented with a ContentDialog

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
  - Closes https://github.com/files-community/Files/issues/11994
  - Closes https://github.com/files-community/Files/issues/8154
  - Closes https://github.com/files-community/Files/issues/7358
  - Closes https://github.com/files-community/Files/issues/11551
  - Closes https://github.com/files-community/Files/issues/11550
  - Closes https://github.com/files-community/Files/issues/11549
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Go to the Security tab of a certain file
   2. See everything is the same as what File Explorer shows
   3. Check if **object owner is shown** correctly.
   4. Check if **object owner can be changed** correctly.
   5. Check if **ACL is shown** correctly.
   6. Check if **an ACE can be removed** correctly.
   7. Check if **an ACE can be added** correctly.
   8. Check if **an ACE can be changed** correctly.

## Screenshots

image

### CODE REVIEW

File: src/Files.App/Views/Properties/SecurityAdvancedPage.xaml [hunk 1]
```diff
@@ -547,72 +412,11 @@
...
-										<StaticResource x:Key="ButtonBackgroundPressed" ResourceKey="SystemControlTransparentBrush" />
-										<StaticResource x:Key="ButtonBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
-									</ResourceDictionary>
-								</DropDownButton.Resources>
-
-								<DropDownButton.Content>
-									<TextBlock Text="{x:Bind InheritanceFlagsHumanized, Mode=OneWay}" TextTrimming="CharacterEllipsis" />
-								</DropDownButton.Content>
-
-								<DropDownButton.Flyout>
```
Comment (yaira2) [inline src/Files.App/Views/Properties/SecurityAdvancedPage.xaml:581]: If the strings are no longer being used, can you remove them from the resw file?

File: src/Files.App/Dialogs/AccessControlEditorDialog.xaml
Comment (yaira2) [inline src/Files.App/Dialogs/AccessControlEditorDialog.xaml:1]: Can you localize the strings?

File: src/Files.App/Views/Properties/SecurityAdvancedPage.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/Views/Properties/SecurityAdvancedPage.xaml:581]: No strings have been unused.

Comment (0x5bfa) [reply src/Files.App/Views/Properties/SecurityAdvancedPage.xaml:581]: Removed unused strings. Thank you for pointint it out!

File: src/Files.App/Dialogs/AccessControlEditorDialog.xaml
Comment (0x5bfa) [reply src/Files.App/Dialogs/AccessControlEditorDialog.xaml:1]: Done!

**Comment by 0x5bfa** [comment]
DONE

**Pull request closed without merging**